### DISCUSSION TITLE
Attempt to detect portion size

### DISCUSSION THREAD

**Discussion by teolemon** [openfoodfacts/robotoff#1195] [Ideas]
### What

- Attempt to detect portion size

### Why

- It's important for nutrition trackers
- It's important for proper nutrition extraction as a secondary signal
- It's important for US users who are used to USDA nutrition labels with portions

### France
![image](https://user-images.githubusercontent.com/1689815/75878980-ce359480-5e1a-11ea-856b-9f2ee790a834.png)
TODO : find relevant regexes
``` 
\nCe sachet contient 5 portions. Uneportion-30g\n
```
https://static.openfoodfacts.org/images/products/333/697/170/3228/3.jpg
https://static.openfoodfacts.org/images/products/333/697/170/3228/3.json

### United States
https://us.openfoodfacts.org/images/products/001/111/063/4627/3.json
https://us.openfoodfacts.org/images/products/001/111/063/4627/3.jpg

**Comment by CloCkWeRX** [comment]
From #218

While extracting full nutrition tables may not be easy, many do feature a serving size label.

Similar to extracting a general quantity, from images identified as nutrition tables; consider looking for
"Serving size 25g" (Australian labels)
"Serving size 125ml" (Australian labels)
"Serving size 2/3 cup (55g)" (Us labels)

I'm not sure about other labelling standards.

**Comment by CloCkWeRX** [comment]
Based on https://world.openfoodfacts.org/state/photos-validated/countries ; I might spend a little bit of time sampling products and adding "dry" and "liquid" example strings to 

https://docs.google.com/spreadsheets/d/1lGiHlthC4wFYLq9n0g8-nA3xXbsdMGfHWNzQ2iB2PKg/edit#gid=0
